lib/irb/completion.rb

```diff
@@ -459,12 +439,12 @@ def retrieve_completion_data(input, bind:, doc_namespace:)
             eval("#{perfect_match_var}.class.name", bind) rescue nil
           else
             candidates = (bind.eval_methods | bind.eval_private_methods | bind.local_variables | bind.eval_instance_variables | bind.eval_class_constants).collect{|m| m.to_s}
-            candidates |= ReservedWords
+            candidates |= RubyLex::RESERVED_WORDS.map(&:to_s)
```

Comment:
I think using `name` will prevent recreating string objects every time this is called:

```
irb(main):002> :foo.name.object_id
=> 11536
irb(main):003> :foo.name.object_id
=> 11536
irb(main):004> :foo.name.object_id
=> 11536
irb(main):005> :foo.to_s.object_id
=> 15608
irb(main):006> :foo.to_s.object_id
=> 16096
```

Reply:
`Symbol#name` is added in Ruby 3.0 and IRB supports >= 2.7